doc/nrf/libraries/bluetooth_services/rpc.rst

```diff
@@ -76,3 +77,26 @@ Then, you can invoke build command like this:
    :class: highlight
 
    west build -b *board* -- -DCONFIG_OVERLAY=my_overlay_file.conf
+
+API documentation
+*****************
+
+This library does not define a new API.
+Instead, it uses Zephyr's Bluetooth API.
+The |NCS| currently supports serialization of:
```

Comment:
```suggestion
The |NCS| currently supports serialization of the following:
```
Personally, I would rather avoid the phrase "the following", but it is the company policy to have that.